API/DTOs/TournamentStatsDTO.cs

```diff
@@ -13,12 +16,12 @@ public class PlayerTournamentStatsDTO
     /// <summary>
     /// List of best tournament performances for the player
     /// </summary>
-    public IEnumerable<PlayerTournamentMatchCostDTO> BestPerformances { get; set; } =
+    public IEnumerable<PlayerTournamentMatchCostDTO> BestPerformances { get; init; } =
```

Comment:
Does init play nice with EF Core? Should we be using this everywhere? Otherwise, I'd stick to `set;` for consistency with other DTOs.